impeller/entity/contents/content_context.h

```diff
@@ -823,7 +823,8 @@ class ContentContext {
 
   bool is_valid_ = false;
   std::shared_ptr<Tessellator> tessellator_;
-  std::shared_ptr<GlyphAtlasContext> glyph_atlas_context_;
+  std::shared_ptr<GlyphAtlasContext> alpha_glyph_atlas_context_;
+  std::shared_ptr<GlyphAtlasContext> color_glyph_atlas_context_;
```

Comment:
I could put these into a map or something else. Feels weird to make a second container but might as well. I don't think we'll end up with a third glyph atlas, at least not for a while